### PR TITLE
Something for #41 (ghci-options configuration)

### PR DESCRIPTION
**Pull request by lwm**
Uhmmm, aside from `vint` telling me this looks OK, I'm not really sure how to test this otherwise. 

This is my first stab at the big bad world of Vim script :baby:

### CODE REVIEW

**Comment by lwm** [comment]
Actually, if someone could tell me how to test this locally, it'd be rad! I was thinking of using the local Git repo syntax for Vundle, like: `Plugin 'file:///home/lwm/path/to/plugin'`. You all do the same or?

**Comment by parsonsmatt** [comment]
Typically when I'm developing I'll set the line to be

```
Plug '~/Projects/intero-neovim'
```

Then it will pull from the local project. I'll make sure it works with the example project first, then try it out on a few bigger projects.

File: autoload/intero/process.vim [hunk 1]
```diff
@@ -219,10 +219,15 @@ function! s:start_buffer(height) abort
     " ID of the buffer.
     exe 'below ' . a:height . ' split'
 
+    let l:invocation = 'ghci --with-ghc intero'
+    if !exists('g:intero_ghci_options')
```
Comment (parsonsmatt) [inline autoload/intero/process.vim]: I think you flipped a bit here.

Comment (lwm) [inline autoload/intero/process.vim]: Ah, woops!

**Comment by lwm** [comment]
> I'll make sure it works with the example project first, then try it out on a few bigger projects.

Righto, will do then and see how it goes.

**Comment by lwm** [comment]
OK, seems to work!

File: example/stack.yaml [hunk 2]
```diff
@@ -1,67 +1,5 @@
...
-# arch: i386
-# arch: x86_64
-# 
-# Extra directories used by stack for building
-# extra-include-dirs: [/path/to/dir]
-# extra-lib-dirs: [/path/to/dir]
-# 
-# Allow a newer minor version of GHC than the snapshot specifies
-# compiler-check: newer-minor
+- 'intero-0.1.20'
```
Comment (rdnetto) [inline example/stack.yaml]: This is the same version in lts-8.23,  so we shouldn't need this line anymore.

Comment (lwm) [inline example/stack.yaml]: Righto, will fix.

**Comment by rdnetto** [comment]
Awesome, thanks for the PR. :)

**Pull request merged**